### PR TITLE
Timesheet: generic improvements

### PR DESCRIPTION
**Pull request by Williambraecky**
The timesheet_grid icon changes when enterprise is present. This changes
the icon from base to the one from enterprise.

Generic improvements of the timesheet app:
- Adds pivot and graph view to 'My Timesheets'
- Adds a graph view to 'All Timesheets'
- Rename 'Timesheet' menu to 'Timesheets'
- Remove unused submenu for configuration, directly leads to settings
  now.

Task ID: 2531317
Relates to odoo/enterprise#19101

### CODE REVIEW

**Review by tlibioulle — COMMENTED** [review]
Good job! :)

Just a single remark, but nothing blocking for now.

File: addons/hr_timesheet/views/res_config_settings_views.xml [hunk 1]
```diff
@@ -118,6 +115,6 @@
         <field name="context">{'module' : 'hr_timesheet', 'bin_size': False}</field>
     </record>
 
-    <menuitem id="hr_timesheet_config_settings_menu_action" name="Settings" parent="hr_timesheet_menu_configuration"
-        action="hr_timesheet_config_settings_action" sequence="0" groups="base.group_system"/>
+    <menuitem id="hr_timesheet_menu_configuration" name="Configuration" parent="timesheet_menu_root"
+        action="hr_timesheet_config_settings_action" groups="group_timesheet_manager,base.group_system" sequence="100"/>
```
[the remark this answers is not in the document]
Comment (Williambraecky) [reply addons/hr_timesheet/views/res_config_settings_views.xml:119]: Right, I wanted to keep the same required groups but this indeed doesn't keep the same behaviour as before :)

**Review by IT-Ideas — APPROVED** [review]
LGTM 🙂 
Thanks for your work 👍🏼

**Comment by IT-Ideas** [comment]
Can you maybe still just mention the PR number in your commits?
When you have more than one commit in the PR (if you have only one the it will be done automatically during merge) it is best to mention it at the bottom of the commit message like:
```txt
Closes #PRNUMBER
```

See [guidelines](https://www.odoo.com/documentation/14.0/fr/developer/reference/guidelines.html#commit-message-structure)

Thanks

**Comment by IT-Ideas** [comment]
As per guideline, can you also rephrase your commit Title with a sentence that starts with a verb?
See [guidelines](https://www.odoo.com/documentation/14.0/fr/developer/reference/guidelines.html#commit-message-header)

Thanks 🙂

**Comment by Williambraecky** [comment]
Hi @IT-Ideas should be good now 😄  let me know if you want anything else changed!

**Comment by IT-Ideas** [comment]
LGTM Thanks 🙂

**Comment by IT-Ideas** [comment]
@robodoo r+

**Comment by IT-Ideas** [comment]
@robodoo rebase-ff

**Pull request closed without merging**